portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,25 +31,43 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
- ArrayList<String> comments = new ArrayList<String>();
+;
     
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Do you want to add a code comment for what this block of code generally is intended to do? It's generally a good coding practice.